.golangci.yml

```diff
@@ -55,7 +55,6 @@ linters:
   disable-all: true
   enable:
     - bodyclose
-    - depguard
```

Comment:
Depguard has become extremely strict to the point of needing an allowlist for all packages if I understand it right. I think we shouldn't bother for the time being, and there's other dependency solutions that could be better if we care in future.